packages/plugin-core/src/KeybindingUtils.ts

```diff
@@ -1,206 +1,330 @@
+      "",
+      `Use [Default Keybindings](${defaultKeybindingJSONCommandUri}) to reference all default keybindings.`,
+      "",
+      "***",
+      "## List of Keybinding Conflicts",
+      conflicts
+        .map((conflict) => {
+          const conflictKeybindingEntry = defaultKeybindingJSON.find(
+            (keybinding) => {
+              return keybinding.command === conflict.commandId;
+            }
+          );
+          const dendronKeybindingEntry = defaultKeybindingJSON.find(
+            (keybinding) => {
+              return keybinding.command === conflict.conflictsWith;
+            }
+          );
+          if (conflictKeybindingEntry === undefined) {
+            return undefined;
+          }
+          if (dendronKeybindingEntry === undefined) {
+            return undefined;
+          }
 
-    // add override if there isn't.
-    const OVERRIDE_EXPAND_LINE_SELECTION = {
-      key: `ctrl+l`,
-      command: "-extension.vim_navigateCtrlL",
-    };
+          const disableBlock = KeybindingUtils.generateKeybindingBlockForCopy({
+            entry: conflictKeybindingEntry,
+            disable: true,
+          });
 
-    const newKeybindings = assign(
-      keybindings,
-      keybindings.concat(OVERRIDE_EXPAND_LINE_SELECTION)
+          const remapDendronBlock =
+            KeybindingUtils.generateKeybindingBlockForCopy({
+              entry: dendronKeybindingEntry,
+              disable: false,
+            });
+
+          const remapConflictBlock =
+            KeybindingUtils.generateKeybindingBlockForCopy({
+              entry: conflictKeybindingEntry,
+              disable: false,
+            });
+
+          const copyCommandUri = (args: { text: string }) =>
+            `command:dendron.copyToClipboard?${encodeURIComponent(
+              JSON.stringify(args)
+            )}`;
+
+          const out = [
+            `### \`${conflict.commandId}\``,
+            `- key: \`${conflictKeybindingEntry.key}\``,
+            `- command: \`${conflict.commandId}\` [Disable](${copyCommandUri({
+              text: disableBlock,
+            })}) / [Remap](${copyCommandUri({ text: remapConflictBlock })})`,
+            `- from: \`${conflict.extensionId}\``,
+            `- conflicts with: \`${
+              conflict.conflictsWith
+            }\` [Remap](${copyCommandUri({ text: remapDendronBlock })})`,
+            "",
+          ].join("\n");
+
+          return out;
+        })
+        .filter((line) => line !== undefined)
+        .join("\n"),
+    ].join("\n");
+
+    const panel = vscode.window.createWebviewPanel(
+      "keybindingConflictPreview",
+      "Keybinding Conflicts",
+      vscode.ViewColumn.Beside,
+      {
+        enableCommandUris: true,
+      }
     );
-    return { data: { keybindingConfigPath, newKeybindings } };
+    panel.webview.html = md.render(contents);
```

Comment:
instead of using `md.render`, can we use the dendron preview and render a note? see [[Using the Dendron Preview on arbitrary markdown|dendron://dendron.docs/pkg.plugin-core.dev.cook#using-the-dendron-preview-on-arbitrary-markdown]]

Reply:
Last time we discussed this we established that we currently can't do this as described in that note.

The method described can be used to render arbitrary markdown from a file that exists outside of a Dendron vault.

I can create another task to figure out a way to make pseudo-NoteProps for this purpose, but I would say it is out of scope for this PR.

Reply:
@SeriousBug I remember you worked on the non-note rendering logic for preview. Is there something I'm missing here about passing arbitrary markdown to our preview? My understanding is that we still need a file to create a pseudo-note for the purpose of rendering.

Reply:
Assuming passing in arbitrary markdown without having it anywhere in the filesystem works, another problem is that we only have one instance of the preview view in any given session, and we don't want to reuse an existing preview panel to render this content. I'm not sure if we can dynamically register multiples of the same views once the extension is activated.